etcd.go

```diff
@@ -167,8 +128,7 @@ func (s *Service) List(ctx context.Context, key string) ([]string, error) {
 		clientv3.WithPrefix(),
```

Comment:
I think problem with tests is here. Remove `clientv3.WithKeysOnly()` option and try again.

Reply:
Indeed, thx!
```
$ go test -tags=integration $(glide novendor)
ok  	github.com/giantswarm/etcdstorage	0.210s
```